match/new_orleans_pd.py

```diff
@@ -205,6 +205,42 @@ def match_stop_and_search_to_pprr(sas, pprr_ipm):
     return sas
 
 
+def match_brady_to_cprr(brady, pprr_ipm):
+    dfa = brady[["uid", "first_name", "last_name"]]
+    dfa.loc[:, "fc"] = dfa.first_name.fillna("").map(lambda x: x[:1])
+    dfa.loc[:, "lc"] = dfa.last_name.fillna("").map(lambda x: x[:1])
+    dfa = dfa.drop_duplicates(subset=["uid"]).set_index("uid")
+
+    dfb = pprr_ipm[["uid", "first_name", "last_name"]]
+    dfb.loc[:, "fc"] = dfb.first_name.fillna("").map(lambda x: x[:1])
+    dfb.loc[:, "lc"] = dfb.last_name.fillna("").map(lambda x: x[:1])
+    dfb = dfb.drop_duplicates(subset=["uid"]).set_index("uid")
+
+    matcher = ThresholdMatcher(
+        ColumnsIndex(["fc", "lc"]),
+        {
+            "first_name": JaroWinklerSimilarity(),
+            "last_name": JaroWinklerSimilarity(),
+        },
+        dfa,
+        dfb,
+        show_progress=True,
+    )
+    decision = 1
+
+    matcher.save_pairs_to_excel(
+        data_file_path(
+            "match/nopd_brady_2021_v_pprr_new_orleans_pd_1946_2018.xlsx"
+        ),
+        decision,
+    )
+    matches = matcher.get_index_pairs_within_thresholds(lower_bound=decision)
+    match_dict = dict(matches)
+
+    sas.loc[:, "uid"] = sas.uid.map(lambda x: match_dict.get(x, x))
+    return sas
```

Comment:
I think `sas` is not defined in this function.

Reply:
If you decide to match brady list's `uid` to post's `uid` then this script is not doing that.